src/components/Accordion/Accordion.tsx

```diff
@@ -170,22 +178,28 @@ class Accordion extends AutoControlledComponent<Extendable<IAccordionProps>, any
     const { panels, renderContent, renderTitle } = this.props
 
     _.each(panels, (panel, index) => {
-      const { content, title } = panel
+      const { content, title, accordion } = panel
       const active = this.isIndexActive(index)
 
       children.push(
         AccordionTitle.create(title, {
           defaultProps: { active, index },
+          generateKey: true,
           overrideProps: this.handleTitleOverrides,
           render: renderTitle,
         }),
       )
       children.push(
         AccordionContent.create(content, {
           defaultProps: { active },
+          generateKey: true,
           render: renderContent,
         }),
       )
+
+      if (active && accordion) {
+        children.push(Accordion.create(accordion))
```

Comment:
Maybe give ```marginLeft``` style to the nested accordion to have a better visual representation.

```jsx
children.push(Accordion.create(accordion, {
    defaultProps: { styles: {
        marginLeft: pxToRem(30),
    }},
}))
```
![accordion stardust](https://user-images.githubusercontent.com/11490517/46649594-401e9580-cbb7-11e8-8c1c-dff4a92bab21.png)

Reply:
We will not hard code the styles in the component. We will do it when consuming it.